web_src/css/repository.css

```diff
@@ -2740,6 +2740,12 @@
   width: 100% !important;
   max-width: 100% !important;
   margin: 0 !important;
+  padding: 0 !important;
```

Comment:
I would also suggest to revert this change.

https://github.com/go-gitea/gitea/pull/23750#issuecomment-1486249539

It causes unnecessary overwriting problems.